### PR TITLE
Fix for Rails 6.1, spec_name -> name.

### PR DESCRIPTION
**Pull request by huacnlee**
https://github.com/rails/rails/pull/38536

### CODE REVIEW

File: lib/pghero.rb [hunk 1]
```diff
@@ -217,13 +219,11 @@ def each_database
...
-        rescue => e
-          puts "#{e.class.name}: #{e.message}"
-          puts
-          first_error ||= e
-        end
+        yield database
+      rescue StandardError => e
+        puts "#{e.class.name}: #{e.message}"
+        puts
+        first_error ||= e
```
Comment (huacnlee) [inline lib/pghero.rb:226]: Sorry, this is autoformat

File: lib/pghero.rb [hunk 2]
```diff
@@ -54,17 +54,17 @@ class << self
...
+                   :duplicate_indexes, :enable_query_stats, :explain, :historical_query_stats_enabled?, :index_caching,
+                   :index_hit_rate, :index_usage, :indexes, :invalid_constraints, :invalid_indexes, :kill, :kill_all, :kill_long_running_queries,
+                   :last_stats_reset_time, :long_running_queries, :maintenance_info, :missing_indexes, :query_stats,
+                   :query_stats_available?, :query_stats_enabled?, :query_stats_extension_enabled?, :query_stats_readable?,
+                   :rds_stats, :read_iops_stats, :region, :relation_sizes, :replica?, :replication_lag, :replication_lag_stats,
+                   :reset_query_stats, :reset_stats, :running_queries, :secret_access_key, :sequence_danger, :sequences, :settings,
+                   :slow_queries, :space_growth, :ssl_used?, :stats_connection, :suggested_indexes, :suggested_indexes_by_query,
+                   :suggested_indexes_enabled?, :system_stats_enabled?, :table_caching, :table_hit_rate, :table_stats,
+                   :total_connections, :transaction_id_danger, :unused_indexes, :unused_tables, :write_iops_stats
```
Comment (huacnlee) [inline lib/pghero.rb:67]: Autoformat via Rubocop

**Comment by ankane** [comment]
Hey @huacnlee, thanks for the PR! I'd like to wait until an RC is out before merging more Rails 6.1 work, but here are a few thoughts on the code:

1. The change will need to be backwards compatible w/ Rails 6.0
2. `spec_name` is also referenced in another file
3. Please leave out the formatting changes

**Comment by huacnlee** [comment]
@ankane Hi, I just updated the code, and added backwards compatible with Rails 6.0

**Comment by ankane** [comment]
Thanks @huacnlee, looks great! Will test once an RC is out.

**Comment by ankane** [comment]
Works great, thanks @huacnlee!

**Pull request merged**